src/hashtable.h

```diff
@@ -42,6 +42,11 @@ typedef uint64_t hashtableIterator[5];
 typedef uint64_t hashtablePosition[2];
 typedef uint64_t hashtableIncrementalFindState[5];
 
+typedef enum {
+    ENTRY_VALID = 0,
+    ENTRY_INVALID
```

Comment:
ENTRY_NONEXISTENT feels more self-explanatory

Reply:
I am not sure that NON-EXISTENT is the accurate phrase, maybe ENTRY_MASK?